pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -71,6 +71,7 @@ type Route struct {
 type Receiver struct {
 	Name             string            `json:"name"`
 	PagerDutyConfigs []PagerDutyConfig `json:"pagerDutyConfigs,omitempty"`
+	WebhookConfigs   []WebhookConfig   `yaml:"webhook_configs,omitempty" json:"webhook_configs,omitempty"`
```

Comment:
out of curiosity: why do we have both `yaml` and `json` descriptors here?

Reply:
thnx, will fix this